ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/CreateProjectAction.java

```diff
@@ -30,15 +30,15 @@
  * @author Dmitry Shnurenko
  */
 @Singleton
-public class NewProjectAction extends AbstractPerspectiveAction {
+public class CreateProjectAction extends AbstractPerspectiveAction {
 
     private final ProjectWizardPresenter wizard;
     private final AnalyticsEventLogger   eventLogger;
     private final AppContext             appContext;
 
     @Inject
-    public NewProjectAction(Resources resources, ProjectWizardPresenter wizard, AnalyticsEventLogger eventLogger, AppContext appContext) {
-        super(Arrays.asList(PROJECT_PERSPECTIVE_ID), "Project...", "Create new project", null, resources.newProject());
+    public CreateProjectAction(Resources resources, ProjectWizardPresenter wizard, AnalyticsEventLogger eventLogger, AppContext appContext) {
+        super(Arrays.asList(PROJECT_PERSPECTIVE_ID), "Create Project...", "Create new project", null, resources.newProject());
```

Comment:
Sorry guys, I have no more time for refactoring commands. I hope each coder will write perfect and well commented code.